hugegraph-api/src/main/java/com/baidu/hugegraph/api/profile/GraphsAPI.java

```diff
@@ -336,6 +336,7 @@ public String flush(@Context GraphManager manager,
         E.checkArgument(readMode != null,
                         "Graph-read-mode can't be null");
         HugeGraph g = graph(manager, graphSpace, graph);
+        manager.graphReadMode(graphSpace, graph, readMode);
```

Comment:
only write back to etcd is not enough. Also should notify other hugegraphserver for modify
or
read also get value from etcd

Reply:
done